src/lib.rs

```diff
@@ -38,8 +38,8 @@
 //! ld.detach().unwrap();
 //! ```
 use crate::bindings::{
-    loop_info64, LOOP_CLR_FD, LOOP_CTL_GET_FREE, LOOP_SET_CAPACITY, LOOP_SET_FD, LOOP_SET_STATUS64,
-    LO_FLAGS_AUTOCLEAR, LO_FLAGS_PARTSCAN, LO_FLAGS_READ_ONLY,
+    loop_config, loop_info64, LOOP_CLR_FD, LOOP_CONFIGURE, LOOP_CTL_GET_FREE, LOOP_SET_CAPACITY,
```

Comment:
Using `LOOP_CONFIGURE` will cause problems with toolchains that do *not* contain that define and the libc headers are generated from a kernel prior `LOOP_CONFIGURE`. The [devicemapper](https://github.com/stratis-storage/devicemapper-rs/blob/master/build.rs) crate had similar problems with features added over the years. 

`LOOP_CONFIGURE` is great but must be guarded by a feature. 
I'm not sure if it's possible to reliably detect the kernel, libc header version etc... from a build.rs.

Reply:
Just checked a build for `aarch64-linux-android` with `cross`:

```
~/loopdev ‹loop-configure› cross build --target=aarch64-linux-android
   Compiling memchr v2.5.0
   Compiling glob v0.3.0
   Compiling libc v0.2.137
   Compiling proc-macro2 v1.0.47
   Compiling minimal-lexical v0.2.1
   Compiling quote v1.0.21
   Compiling cfg-if v1.0.0
   Compiling unicode-ident v1.0.5
   Compiling bindgen v0.60.1
   Compiling regex-syntax v0.6.27
   Compiling lazycell v1.3.0
   Compiling bitflags v1.3.2
   Compiling shlex v1.1.0
   Compiling peeking_take_while v0.1.2
   Compiling rustc-hash v1.1.0
   Compiling lazy_static v1.4.0
   Compiling libloading v0.7.3
   Compiling clang-sys v1.4.0
   Compiling nom v7.1.1
   Compiling errno v0.2.8
   Compiling regex v1.6.0
   Compiling cexpr v0.6.0
   Compiling loopdev v0.5.0 (/project)
error[E0432]: unresolved imports `crate::bindings::loop_config`, `crate::bindings::LOOP_CONFIGURE`
  --> src/lib.rs:41:5
   |
41 |     loop_config, loop_info64, LOOP_CLR_FD, LOOP_CONFIGURE, LOOP_CTL_GET_FREE, LOOP_SET_CAPACITY,
   |     ^^^^^^^^^^^                            ^^^^^^^^^^^^^^ no `LOOP_CONFIGURE` in `bindings`
   |     |
   |     no `loop_config` in `bindings`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `loopdev` due to previous error
```

I don't know the exact version of the libc headers used in the NDK here but it's notorious outdated ;-)